.github/workflows/ci.yml

```
@@ -2,29 +2,49 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
   pull_request:
-    branches: [ master ]
+  schedule:
```

Let's make those comments a bit more helpful and explain *why* the constructs are used.

```suggestion
  # Run CI daily even without code changes to ensure tests pass with 
  # updated dependencies.
  schedule:
```